src/geowombat/backends/xarray_.py

```diff
@@ -227,7 +210,7 @@ def warp_open(
     resampling='nearest',
     dtype=None,
     netcdf_vars=None,
-    nodata=0,
+    nodata=None,

```

Comment:
New default is `None`, or take the nodata value from the file.

Reply:
Yeah I think that is a good idea.